src/pages/static/assets/global.css

```
@@ -51,18 +51,52 @@ html {
 }
 
 /* EDITOR CONTENT STYLES */
-.toastui-editor-main * {
+.toastui-editor-contents * {
   font-family: poppins, sans-serif !important;
 }
-.toastui-editor-main h1,
-.toastui-editor-main h2,
-.toastui-editor-main h3 {
+.toastui-editor-contents h1,
+.toastui-editor-contents h2,
+.toastui-editor-contents h3,
+.toastui-editor-contents h4,
+.toastui-editor-contents h5,
+.toastui-editor-contents h6 {
```

- You can shorten this kind of code this way
```suggestion
.toastui-editor-contents :where(h1, h2, h3, h4, h5, h6) {
```
- I think the where selector has higher specifity, so whenever you use it, you won't need to use `!important` test that though.

---

This is awesome ! thanks